epochX/cudacpp/CODEGEN/PLUGIN/CUDACPP_SA_OUTPUT/model_handling.py

```diff
@@ -736,7 +745,12 @@ def write_hardcoded_parameters(self, params):
             type, pars = line.rstrip(';').split(' ') # strip trailing ';'
             for par in pars.split(','):
                 ###print(len(pardef_lines), par) # for debugging
-                pardef_lines[par] = ( 'constexpr ' + type + ' ' + par )
+                if par in majorana_widths:
+                    pardef_lines[par] = ( 'constexpr ' + type + ' ' + par + "_abs" )
+                elif par in deviceparams:
```

Comment:
Can you comment on the reason of handling majorana width in a special way? I'm just puzzled by it